components/copy_url_context_menu/copy_url_context_menu.jsx

```diff
@@ -51,9 +51,12 @@ export default class CopyUrlContextMenu extends React.Component {
         );
 
         const contextMenuTrigger = (
-            <ContextMenuTrigger id={'copy-url-context-menu' + this.props.menuId}>
+            <ContextMenuTrigger
+                id={'copy-url-context-menu' + this.props.menuId}
+                holdToDisplay={-1}
+            >
                 {this.props.children}
-            </ContextMenuTrigger>
+            </ContextMenuTrigger >
```

Comment:
Looks like a space snuck in here. 😛

Reply:
I was testing a code formatter and I have to ditch it for things like this :( saw the other problems but not this one, thanks for spotting it